README.md

```diff
@@ -19,7 +19,7 @@ You can also generate and install modules manually.
 
 ### Install via pip package
 
-fake-bpy-module is registered to PyPI.  
```

Comment:
Please do not delete last spaces.
The last spaces is needed to render HTML correctly on browser.

Reply:
sorry, this was VSCode auto-formatter

Reply:
reverted

Reply:
@nutti out of interest: Which parser does output invalid HTML without the space on markdown? 
On Github, the output looked fine: https://github.com/nutti/fake-bpy-module/blob/33804ca7bb2496406299d051faa22e20d8da13bd/README.md

Reply:
@grische

Thanks!
It's weired behaviour. In my environment, your markdown looks like.

![github_20200617_1](https://user-images.githubusercontent.com/1463952/84841453-50219480-b07d-11ea-8c88-e58bf4019c62.png)

Expected behaviour is:

![github_20200617_2](https://user-images.githubusercontent.com/1463952/84841473-5d3e8380-b07d-11ea-8be5-ffaccb08e7e8.png)

Anyway, thanks for fix to this.